### PR TITLE
PIM-5095 : Add Xlsx Import for groups and variant groups

### PR DESCRIPTION
**Pull request by philippemossiere**
Add Xlsx Import for groups and variant groups

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N
| Added Behats                      | Y
| Changelog updated                 | N
| Review and 2 GTM                  | 
| Micro Demo to the PO (Story only) | 
| Migration script                  | N
| Tech Doc                          | N

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

File: features/Context/catalog/footwear/jobs.yml [hunk 1]
```diff
@@ -81,16 +81,24 @@ jobs:
             enclosure:  '"'
             withHeader: true
             filePath: /tmp/association_type.csv
-    footwear_group_import:
+    csv_footwear_group_import:
         connector: Akeneo CSV Connector
         alias:     csv_group_import
-        label:     Footwear group import
+        label:     CSV Footwear group import
```
Comment (jjanvier) [inline features/Context/catalog/footwear/jobs.yml]: minus f to footwear everywhere ;)

File: features/import/delete_an_import.feature [hunk 2]
```diff
@@ -9,16 +9,31 @@ Feature: Delete import
...
-  Scenario: Successfully cancel the deletion of an import job
-    Given I cancel the deletion
-    Then the grid should contain 9 elements
+  Scenario: Successfully cancel the deletion of a CSV import job
+    Given I delete the "csv_footwear_product_import" job
+    And I cancel the deletion
+    Then the grid should contain 11 elements
     And I should see import profile "csv_footwear_product_import"
+
+  Scenario: Successfully delete a XLSX import job
```
Comment (jjanvier) [inline features/import/delete_an_import.feature]: do we really need these Behat?

File: features/import/delete_an_import.feature [hunk 3]
```diff
@@ -9,16 +9,31 @@ Feature: Delete import
     Given a "footwear" catalog configuration
     And I am logged in as "Peter"
     And I am on the imports page
-    Then the grid should contain 9 elements
-    When I delete the "csv_footwear_product_import" job
+    And I change the page size to 25
+    Then the grid should contain 11 elements
```
Comment (phaseinducer) [inline features/import/delete_an_import.feature]: Weird to assert something in the background.

**Comment by jjanvier** [comment]
gtm

File: features/import/delete_an_import.feature [hunk 4]
```diff
@@ -9,16 +9,31 @@ Feature: Delete import
...
 
-  Scenario: Successfully delete an import job
-    Given I confirm the deletion
+  Scenario: Successfully delete a CSV import job
+    Given I delete the "csv_footwear_product_import" job
+    And I confirm the deletion
```
Comment (phaseinducer) [inline features/import/delete_an_import.feature]: When

File: features/import/delete_an_import.feature [hunk 5]
```diff
@@ -9,16 +9,31 @@ Feature: Delete import
...
-    And the grid should contain 8 elements
+    And the grid should contain 10 elements
     And I should not see import profile "csv_footwear_product_import"
 
-  Scenario: Successfully cancel the deletion of an import job
-    Given I cancel the deletion
-    Then the grid should contain 9 elements
+  Scenario: Successfully cancel the deletion of a CSV import job
+    Given I delete the "csv_footwear_product_import" job
+    And I cancel the deletion
```
Comment (phaseinducer) [inline features/import/delete_an_import.feature]: When

File: features/import/delete_an_import.feature [hunk 6]
```diff
@@ -9,16 +9,31 @@ Feature: Delete import
...
+    Then the grid should contain 11 elements
     And I should see import profile "csv_footwear_product_import"
+
+  Scenario: Successfully delete a XLSX import job
+    Given I delete the "xlsx_footwear_product_import" job
+    And I confirm the deletion
```
Comment (phaseinducer) [inline features/import/delete_an_import.feature]: When

File: features/import/delete_an_import.feature [hunk 7]
```diff
@@ -9,16 +9,31 @@ Feature: Delete import
...
+    Then I should see flash message "Import profile successfully removed"
+    And the grid should contain 10 elements
+    And I should not see import profile "xlsx_footwear_product_import"
+
+  Scenario: Successfully cancel the deletion of a XLSX import job
+    Given I delete the "xlsx_footwear_product_import" job
+    And I cancel the deletion
```
Comment (phaseinducer) [inline features/import/delete_an_import.feature]: When

File: features/import/delete_an_import.feature (hunk 2 above)
Comment (phaseinducer) [inline features/import/delete_an_import.feature]: Not sure it's useful to test this for all types of import profiles.

File: features/import/product_variant_group/xlsx/import_variant_group_with_valid_properties.feature [hunk 8]
```diff
@@ -0,0 +1,52 @@
+@javascript
+Feature: Execute an import with valid properties
```
Comment (skeleton) [inline features/import/product_variant_group/xlsx/import_variant_group_with_valid_properties.feature]: this feature name already exists in `features/import/product_variant_group/import_variant_group_with_valid_properties.feature` file. Please change it

File: features/import/product_group/xlsx/import_groups.feature [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+      AKENEO_XSELL;;RELATED
+      """
+    And the following job "xlsx_footwear_group_import" configuration:
+      | filePath | %file to import% |
+    When I am on the "xlsx_footwear_group_import" import job page
+    And I launch the import job
+    And I wait for the "xlsx_footwear_group_import" job to finish
+    Then I should see "This property cannot be changed"
+    And I should see "read lines 1"
+    And I should see "Skipped 1"
```
Comment (phaseinducer) [inline features/import/product_group/xlsx/import_groups.feature]: Not a big deal but try to be consistent on capitalization.

File: features/import/delete_an_import.feature (hunk 2 above)
Comment (philippemossiere) [inline features/import/delete_an_import.feature]: I can remove it, if you think they are not necessary.

**Comment by phaseinducer** [comment]
Little comments then GTM :+1:

File: features/import/delete_an_import.feature (hunk 3 above)
Comment (philippemossiere) [inline features/import/delete_an_import.feature]: Ok, I'll fix it

**Pull request merged**